### ISSUE TITLE
Allow const uint8_t* parameter to pb_istream_from_buffer()

### ISSUE THREAD

**Issue by GoogleCodeExporter** [nanopb/nanopb#152]
```
Describe the requested feature:
It would be nice if pb_istream_from_buffer() casts away const internally, so 
that one can give it const uint8_t* parameter.

In what situation would the feature be useful:
When trying to maintain const-correctness in the user application.



```

Original issue reported on code.google.com by `Petteri.Aimonen` on 5 Apr 2015 at 6:16

**Comment by PetteriAimonen** [comment]
Fixed by pull request #190

**Comment by PetteriAimonen** [comment]
Fix released in nanopb-0.3.5.
